tests/azure-test/azure_test_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package azuretest
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	testutils "k8s.io/cloud-provider-azure/tests/azure-test/utils"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	//_ "k8s.io/cloud-provider-azure/tests/azure-test/basic"
+	_ "k8s.io/cloud-provider-azure/tests/azure-test/network"
```

Comment:
why there is `_` here?

Reply:
also refer to http://onsi.github.io/ginkgo/
Import the package for test case implement but no usage